extensions/vscode/package.json

```diff
@@ -251,9 +251,22 @@
         "title": "View Content Log on Connect",
         "icon": "$(output)",
         "category": "Posit Publisher"
+      },
+      {
+        "command": "posit.publisher.deployWithEntrypoint",
+        "title": "Deploy with this Entrypoint",
+        "icon": "$(cloud-upload)",
```

Comment:
I tested this out and it does work, and doesn't look too bad even though it is being scaled down from `450x450` to `16x16`

![CleanShot 2024-07-03 at 14 25 44@2x](https://github.com/posit-dev/publisher/assets/19917631/b33af35c-ea43-448a-8473-449100357d5e)

However since it is a `png` it doesn't get colored using the active theme. So even if I changed it to be white/gray/etc it would look incorrect depending on the theme used.

Personally I'd prefer to avoid the non-native look until we get that fixed.

If others disagree, it can be easily changed while I'm out with the below example:

```json
      {
        "command": "posit.publisher.deployWithEntrypoint",
        "title": "Deploy with this Entrypoint",
        "icon": "assets/img/color/posit-publisher.png",
        "category": "Posit Publisher"
      }
```